### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by armandojaleo**
Added prism score property to payment response risk object
Added device_session_id to RiskRequest
Restore first_name and country fields from PaymentRecipient schema
Added mandatory fields for MIT/CIT payments sources

### CODE REVIEW

File: payments/nas/sources/sources.go [hunk 1]
```diff
@@ -20,17 +20,20 @@ type (
...
 	RequestIdSource struct {
-		Type          payments.SourceType `json:"type,omitempty"`
-		Id            string              `json:"id,omitempty"`
-		Cvv           string              `json:"cvv,omitempty"`
-		PaymentMethod string              `json:"payment_method,omitempty"`
+		Type              payments.SourceType `json:"type,omitempty"`
+		Id                string              `json:"id,omitempty"`
+		Cvv               string              `json:"cvv,omitempty"`
+		PaymentMethod     string              `json:"payment_method,omitempty"`
+		Stored            *bool               `json:"stored,omitempty"`
```
Comment (Armando-Ibarra) [inline payments/nas/sources/sources.go:27]: Please also add `Stored` and `StoreForFutureUse` for RequestIdSource ABC

File: payments/nas/sources/sources.go [hunk 2]
```diff
@@ -20,17 +20,20 @@ type (
...
+		StoreForFutureUse *bool               `json:"storeForFutureUse,omitempty"`
 	}
 
 	RequestTokenSource struct {
 		Type              payments.SourceType `json:"type,omitempty"`
 		Token             string              `json:"token,omitempty"`
 		BillingAddress    *common.Address     `json:"billing_address,omitempty"`
 		Phone             *common.Phone       `json:"phone,omitempty"`
+		Stored            *bool               `json:"stored,omitempty"`
```
Comment (Armando-Ibarra) [inline payments/nas/sources/sources.go:36]: Same for token source
Please also add `Stored` and `StoreForFutureUse` for ABC

**Review by Armando-Ibarra — CHANGES_REQUESTED** [review]
Just another addition, we need to have MIT properties for ABC

File: payments/nas/sources/sources.go (hunk 1 above)
Comment (martinseco) [reply payments/nas/sources/sources.go:27]: Are we sure that these fields need to be nullable? Being bool fields and using the omitempty tag, means that if the field is not set to true, we don't parse it when serializing to json @Armando-Ibarra

File: payments/payments.go [hunk 3]
```diff
@@ -280,14 +280,17 @@ type (
...
 	}
 
 	PaymentRecipient struct {
-		DateOfBirth   string `json:"dob,omitempty"`
-		AccountNumber string `json:"account_number,omitempty"`
-		Zip           string `json:"zip,omitempty"`
-		LastName      string `json:"last_name,omitempty"`
+		DateOfBirth   string          `json:"dob,omitempty"`
+		AccountNumber string          `json:"account_number,omitempty"`
+		CountryCode   *common.Country `json:"country_code,omitempty"`
```
Comment (martinseco) [inline payments/payments.go:290]: I don't think this should not be a pointer

**Review by martinseco — CHANGES_REQUESTED** [review]
I think we should further discuss the need to use bool pointers here @Armando-Ibarra @armandojaleo

File: payments/nas/sources/sources.go (hunk 1 above)
Comment (armandojaleo) [reply payments/nas/sources/sources.go:27]: In previous version we don't find this properties in the API reference, but should we add them anyway?

File: payments/payments.go (hunk 3 above)
Comment (armandojaleo) [reply payments/payments.go:290]: 👍

**Review by Armando-Ibarra — APPROVED** [review]

**Review by martinseco — APPROVED** [review]

**Review by martinseco — APPROVED** [review]
Nice!

**Pull request merged**